.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -1,7 +1,7 @@
 ---
   name: ARO HCP Dev Environment Continuous Deployment
   env:
-    REGION: westus3
```

Comment:
most of the makefiles still require REGION over LOCATION
so please leave the REGION var here and add LOCATION on top. we can clean up once we adjusted all Makefiles

Reply:
Ok, sorry, I didn't realize the impact on other components.
I'm trying to contain the scope of this to the RP, so I'll back this change out and leave it as REGION.
The workaround I added earlier for the RP (`LOCATION ?= ${REGION}`) should suffice for now.